setup.cfg

```diff
@@ -24,261 +21,204 @@ indent = '    '
+	unsupported-membership-test,
 	unused-format-string-argument,
-	format-combined-specification,
-	missing-format-attribute,
-	invalid-format-index,
-	anomalous-backslash-in-string,
-	anomalous-unicode-escape-in-string,
-	bad-open-mode,
-	boolean-datetime,
-	fatal,
+	unused-format-string-key,
+	used-before-assignment,
+	using-constant-test,
+	yield-outside-function,
+
 	astroid-error,
-	parse-error,
+	fatal,
 	method-check-failed,
-	django-not-available,
+	parse-error,
 	raw-checker-failed,
-	django-not-available-placeholder,
+
 	empty-docstring,
 	invalid-characters-in-docstring,
 	missing-docstring,
 	wrong-spelling-in-comment,
 	wrong-spelling-in-docstring,
+
+	unused-argument,
 	unused-import,
 	unused-variable,
-	unused-argument,
-	exec-used,
+
 	eval-used,
+	exec-used,
+
 	bad-classmethod-argument,
 	bad-mcs-classmethod-argument,
 	bad-mcs-method-argument,
-	bad-whitespace,
+	bare-except,
+	broad-except,
 	consider-iterating-dictionary,
 	consider-using-enumerate,
+	global-at-module-level,
+	global-variable-not-assigned,
 	literal-used-as-attribute,
+	logging-format-interpolation,
+	logging-not-lazy,
 	multiple-imports,
 	multiple-statements,
-	old-style-class,
+	no-classmethod-decorator,
+	no-staticmethod-decorator,
+	protected-access,
+	redundant-unittest-assert,
+	reimported,
+	simplifiable-if-statement,
 	simplifiable-range,
 	singleton-comparison,
 	superfluous-parens,
 	unidiomatic-typecheck,
-	unneeded-not,
-	wrong-assert-type,
-	simplifiable-if-statement,
-	no-classmethod-decorator,
-	no-staticmethod-decorator,
-	unnecessary-pass,
 	unnecessary-lambda,
-	useless-else-on-loop,
+	unnecessary-pass,
 	unnecessary-semicolon,
-	reimported,
-	global-variable-not-assigned,
-	global-at-module-level,
-	bare-except,
-	broad-except,
-	logging-not-lazy,
-	redundant-unittest-assert,
-	model-missing-unicode,
-	model-has-unicode,
-	model-no-explicit-unicode,
-	protected-access,
-	deprecated-module,
+	unneeded-not,
+	useless-else-on-loop,
+	wrong-assert-type,
+
 	deprecated-method,
+	deprecated-module,
+
+	too-many-boolean-expressions,
 	too-many-nested-blocks,
 	too-many-statements,
-	too-many-boolean-expressions,
-	ungrouped-imports,
+
+	wildcard-import,
 	wrong-import-order,
 	wrong-import-position,
-	wildcard-import,
+
 	missing-final-newline,
 	mixed-line-endings,
 	trailing-newlines,
 	trailing-whitespace,
 	unexpected-line-ending-format,
-	mixed-indentation,
+
+	bad-inline-option,
 	bad-option-value,
+	deprecated-pragma,
 	unrecognized-inline-option,
 	useless-suppression,
-	bad-inline-option,
-	deprecated-pragma,
 disable =
-	bad-continuation,
-	invalid-name,
-	misplaced-comparison-constant,
-	file-ignored,
 	bad-indentation,
-	lowercase-l-suffix,
-	unused-wildcard-import,
-	global-statement,
-	no-else-return,
-	apply-builtin,
-	backtick,
-	basestring-builtin,
-	buffer-builtin,
-	cmp-builtin,
-	cmp-method,
-	coerce-builtin,
-	coerce-method,
-	delslice-method,
-	dict-iter-method,
-	dict-view-method,
+	consider-using-f-string,
```

Comment:
why not?

Reply:
i prefered to keep the pylint default rules, sure seems is better to style or recommend for everything. After this commit [3f252b8](https://github.com/eduNEXT/eox-nelp/pull/25/commits/3f252b85d43928eaf42c235ce6ca27704dd978c6)I started styling the last steps.
![Peek 2023-02-21 15-13](https://user-images.githubusercontent.com/51926076/220454843-63da76aa-7138-4f0e-8b5c-4af44b3654ed.gif)